solutions/usaco-808.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: usaco-808
+source: USACO Bronze 2018 February
+title: Hoofball
+author: Ryan Chou
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_hoofball_bronze_feb18.html)
+
```

Comment:
add a solution header here

Reply:
why?? i think it's unnecessary...

Reply:
i thought we always hadthe header when there was an explanation

Reply:
err i think the header is ## Explanation (under official analysis), see [here](https://usaco.guide/general/adding-solution#steps)

and the explanation is so short, i didn't think we needed the header...

Reply:
Oops ig i meant explanation then, but i think we should still add it for consistency

Reply:
sure, i can just fix it so we can merge